vscode-extension/package.json

```diff
@@ -115,8 +145,11 @@
     "typecheck": "tsc --noEmit",
     "prettier-check": "prettier -c .",
     "lint": "eslint --max-warnings 0 .",
-    "vscode:prepublish": "tsc",
-    "build-local": "vsce package"
+    "vscode:prepublish": "rm -f tsconfig.tsbuildinfo && rm -rf out && yarn run esbuild-base -- --minify",
+    "build-local": "vsce package",
+    "esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=out/extension.js --external:vscode --format=cjs --platform=node",
+    "esbuild": "npm run esbuild-base -- --sourcemap",
+    "esbuild-watch": "npm run esbuild-base -- --sourcemap --watch"
```

Comment:
Pretty much just copied this from the VSCode docs https://code.visualstudio.com/api/working-with-extensions/bundling-extension#using-esbuild